src/devtools/shared/async-storage.ts

```diff
@@ -44,16 +44,16 @@
  *
  */
 
-const DBNAME = "devtools-async-storage";
 const DBVERSION = 1;
 const STORENAME = "keyvaluepairs";
```

Comment:
Also... this seems like an abuse of IndexedDB? We only have one "item" and it's a giant blob? That seems bad.

Reply:
AFAICT, this is like having a postgres table where you just have one column called `data` and it's a JSON column 😵‍💫

Reply:
@jcmorrow this goes back to what i was saying about `state.pendingBreakpoints` - it ends up being a deserialization of _alllll_ breakpoints for _alllll_ recordings :)